### DISCUSSION TITLE
320 kbps

### DISCUSSION THREAD

**Discussion by Ar7hurus** [Shabinder/SpotiFlyer#786] [Q&A]
Hello
Why the app can't download the songs in 320 kbps from Spotify?
I have the highest quality set in the Spotify app as well as 320 kbps in SpotiFlyer and it downloads songs only in 128 kbps.
I'm doing something wrong here?

**Comment by JayR-Git** [comment]
Created an issue from this. I'm having the same problem.

**Comment by tkalenkoyuriy** [comment]
Hey!
Similar problem.
Download only at 128 kbps

**Comment by Shabinder** [comment]
See https://github.com/Shabinder/SpotiFlyer/issues/1015

**Reply by wolfram85** [reply]
in my opinion the problem is may be another one. The app does not look for "premium", I mean, the link generated for instance, in YT Music is for everybody. HQ Version is available only for premium user on server side, that means that probably the Spotiflyer needs to "login" first into YT Music, and than, will be able to download the premium HQ version.
